cob_description/urdf/sensors/sick_s300_laser.gazebo.xacro

```diff
@@ -13,8 +13,8 @@
             <horizontal>
               <samples>541</samples>
               <resolution>1.0</resolution>
-              <min_angle>${min_angle}</min_angle>
```

Comment:
please keep the parameters min_angle and max_angle, the value depends on the robot.
For example for cob4 is +/- 2.094 https://github.com/ipa320/cob_common/pull/174/files#diff-5e8247da45c8c8d6086e53b8f1db26d6 and for cob3 +/- 2.3562  https://github.com/ipa320/cob_common/pull/174/files#diff-36d0d486a88dc3990599c7fcc4ecfc6d